app/templates/components/tabbed-navigation.hbs

```diff
@@ -1,8 +1,8 @@
 {{#if device.isMobile}}
-  <div role="button" class="ui segment center aligned" {{action 'toggleMenu'}}>
+  <div role="button" class="ui segment center aligned" onClick={{action 'toggleMenu'}}>
```

Comment:
We should never shoot in the dark. We need to know what what is happening the way it is happening.